src/plugins/intel_npu/tests/unit/npuw/unpack.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#ifdef __AVX2__
```

Comment:
@ilya-lavrenov is that the right way to detect AVX2, or OpenVINO comes with its own compile definition?

Reply:
@dmatveev i think we should stick to openvino form or forced disabling based on cmake configs in form of   one of macro 

```code
set(_DEFINE_SSE42    "HAVE_SSE42"   ${_DEFINE_ANY})
set(_DEFINE_AVX      "HAVE_AVX"     ${_DEFINE_SSE42})
set(_DEFINE_AVX2     "HAVE_AVX2"    ${_DEFINE_AVX})
set(_DEFINE_AVX512F  "HAVE_AVX512F" ${_DEFINE_AVX2})
```

for our tests,  #ifdef HAVE_AVX2 should be enough, while tests code itself can be rewritten in ref without intrinsics,